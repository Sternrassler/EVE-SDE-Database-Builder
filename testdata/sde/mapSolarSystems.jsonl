{"solarSystemID":30000001,"solarSystemName":"Tanoo","regionID":10000001,"constellationID":20000001,"x":4.5e16,"y":2.0e16,"z":-1.8e17,"security":0.9,"securityClass":"B"}
{"solarSystemID":30000142,"solarSystemName":"Jita","regionID":10000002,"constellationID":20000020,"x":1.2e17,"y":5.0e16,"z":-3.0e16,"security":0.9,"securityClass":"B"}
{"solarSystemID":30000003,"solarSystemName":"H-PA29","regionID":10000003,"constellationID":20000003,"x":-8.0e16,"y":1.5e17,"z":5.0e16,"security":-0.5,"securityClass":"G"}
