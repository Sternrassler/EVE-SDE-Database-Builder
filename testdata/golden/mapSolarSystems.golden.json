[
  {
    "solarSystemID": 30000001,
    "solarSystemName": "Tanoo",
    "regionID": 10000001,
    "constellationID": 20000001,
    "x": 45000000000000000,
    "y": 20000000000000000,
    "z": -180000000000000000,
    "security": 0.9,
    "securityClass": "B"
  },
  {
    "solarSystemID": 30000142,
    "solarSystemName": "Jita",
    "regionID": 10000002,
    "constellationID": 20000020,
    "x": 120000000000000000,
    "y": 50000000000000000,
    "z": -30000000000000000,
    "security": 0.9,
    "securityClass": "B"
  },
  {
    "solarSystemID": 30000003,
    "solarSystemName": "H-PA29",
    "regionID": 10000003,
    "constellationID": 20000003,
    "x": -80000000000000000,
    "y": 150000000000000000,
    "z": 50000000000000000,
    "security": -0.5,
    "securityClass": "G"
  }
]